notifications_utils/clients/redis/bounce_rate.py

```diff
@@ -38,8 +44,24 @@ def set_notifications_seeded(self, service_id: str, seeded_data: dict) -> None:
     def set_hard_bounce_seeded(self, service_id: str, seeded_data: dict) -> None:
         self._redis_client.add_data_to_sorted_set(hard_bounce_key(service_id), seeded_data)
 
-    def get_bounce_rate(self, service_id: str, bounce_window=_twenty_four_hour_window_ms()) -> float:
+    def set_seeding_started(self, service_id: str) -> None:
+        """Set a flag in Redis to indicate that we have started to seed data for a given service"""
+        self._redis_client.set(seeding_started_key(service_id), "True")
+        self._redis_client.expire(seeding_started_key(service_id), TWENTY_FOUR_HOURS_IN_SECONDS)
 
+    def get_seeding_started(self, service_id: str) -> bool:
+        """Returns True if seeding is complete, False otherwise"""
```

Comment:
i dont have a better word for it...but get_seeding_started returning "True" doesn't indicate that its complete... I dont know what you would name it ow though

Reply:
I forgot to update that docstring, I'll make it more clear